app.js

```diff
@@ -0,0 +1,60 @@
+const express = require("express")
+const app = express();
+const mongoose = require("mongoose") 
+const bp = require("body-parser")
+require('dotenv/config')
+// const User = require("../models/User");
+// const Question = require("../models/Question");
+// const Map = require("../models/Map");
+
+app.use(bp.urlencoded({extended: false}))
```

Comment:
use [express's built-in body parser](http://expressjs.com/en/api.html). You can use it like, `express.json()` or `express.urlencoded({extended: false})`.